content/applications/finance/accounting/taxes/avatax.rst

```diff
@@ -1,80 +1,405 @@
+
+Test connection
+~~~~~~~~~~~~~~~
+
+After entering all the above information into the AvaTax setup on Odoo, click :guilabel:`Test
+connection`. This will ensure that the :guilabel:`API ID` and :guilabel:`API KEY` are indeed
+correct and a connection is made between Odoo and the AvaTax application programming interface
+(API).
+
+Sync parameters
+~~~~~~~~~~~~~~~
 
-If preferred, choose to keep the newly validated address or the original address in the wizard that
-pops up.
+Upon finishing the configuration and settings of the AvaTax section click the :guilabel:`Sync
+Parameters` button. This action will synchronize the exemption codes from AvaTax.
 
-.. image:: avatax/choose-customer-address.png
+.. _avatax/fiscal_positions:
+
+Fiscal position
+---------------
+
+Next, navigate to :menuselection:`Accounting app --> Configuration --> Accounting: Fiscal
+Positions`. A :guilabel:`Fiscal Position` is listed called :guilabel:`Automatic Tax Mapping
+(AvaTax)`. Click it to open AvaTax's fiscal position configuration page.
+
+Here, ensure that the :guilabel:`Use AvaTax API` checkbox is ticked.
+
+Optionally, tick the checkbox next to the field labeled: :guilabel:`Detect Automatically`. Should
+this option be ticked, then, Odoo will automatically apply this :guilabel:`Fiscal Position` for
+transactions in Odoo.
+
+Enabling :guilabel:`Detect Automatically` also makes specific parameters, such as :guilabel:`VAT
+required`, :guilabel:`Foreign Tax ID`, :guilabel:`Country Group`, :guilabel:`Country`,
+:guilabel:`Federal States`, or :guilabel:`Zip Range` appear. Filling this parameters will filter the
+:guilabel:`Fiscal Position` usage. Leaving them blank will ensure all calculations are made using
+this :guilabel:`Fiscal Position`.
+
+.. warning::
+   Should the :guilabel:`Detect Automatically` checkbox not be ticked, then each customer will need
+   to have the :guilabel:`Fiscal Position` set on their :guilabel:`Sales and Purchase` tab of the
+   contact record. To do so, navigate to :menuselection:`Sales app --> Order --> Customers` or
+   :menuselection:`Contacts app --> Contacts`. Then select and customer or contact to set the fiscal
+   position on.
+
+   Navigate to the :guilabel:`Sales and Purchase` tab and down to the section labeled
+   :guilabel:`Fiscal Position`. Set the :guilabel:`Fiscal Position` field to the fiscal position
+   for the customer.
+
+.. seealso::
+   :doc:`fiscal_positions`
+
+AvaTax accounts
+~~~~~~~~~~~~~~~
+
+Upon selecting the checkbox option for :guilabel:`Use AvaTax API` a new :guilabel:`AvaTax` tab
+appears. Click into this tab to reveal two different settings.
+
+The first setting is the :guilabel:`AvaTax Invoice Account`, while the second is, :guilabel:`AvaTax
+Refund Account`. Ensure both accounts are set for smooth end-of-year record keeping. Consult a
+certified public accountant (CPA) for specific guidance on setting both accounts.
+
+Click :guilabel:`Save` to implement the changes.
```

Comment:
```suggestion

```
With auto-save, I don't believe this line is necessary

Reply:
This document is written for Odoo 15